glossary.yml

```diff
@@ -3162,6 +3217,9 @@
       Often a programming error.
 
 
+- slug: inheritance
```

Comment:
```suggestion
```

I don't see anywhere you've linked to this that would require adding an empty slug, so please don't do so. As we do not currently have this definition, I will add it to the list of words we need defined, instead.

Reply:
Sorry this may be half-baked. I mentioned "継承" (inheritance) in the Japanese definition of class but decided not to link to it since it's not defined yet. I'm actually surprised it's not been covered since "parent" and "child" classes have already been included: https://github.com/carpentries/glosario/pull/94 

I noticed other slugs were not defined: "degrees_of_freedom", "masking, "shiny". Is there a reason for this?

I'm okay to remove this slug for now and define it elsewhere. The Japanese definition of class would be compatible with adding a link to it when it's ready.